R/end_rains.R

```diff
@@ -62,38 +60,38 @@ end_rains <- function(data, date_time, station = NULL, year = NULL, rain = NULL,
     year_type <- data_book$get_column_data_types(data_name=data, columns=year)
     data_book$convert_column_to_type(data_name=data, col_names=year, to_type="factor")
     data_book$convert_linked_variable(from_data_frame=data, link_cols=c(year))
-    roll_sum_rain <- instatCalculations::instat_calculation$new(type="calculation", function_exp="RcppRoll::roll_sumr(x=rain, n = interval_length, fill=NA, na.rm=FALSE)", result_name="roll_sum_rain", calculated_from=setNames(list(rain), data))
-    conditions_filter <- instatCalculations::instat_calculation$new(type="filter", function_exp="(roll_sum_rain > min_rainfall) | is.na(x=roll_sum_rain)", sub_calculations=list(roll_sum_rain))
+    roll_sum_rain <- instatCalculations::instat_calculation$new(type="calculation", function_exp=paste0("RcppRoll::roll_sumr(x=rain, n = ", interval_length, ", fill=NA, na.rm=FALSE)"), result_name="roll_sum_rain", calculated_from=setNames(list(rain), data))
```

Comment:
It was having an issue since it couldn't read what `interval_length` was. Because this is a parameter with a value, we add in a `paste0` around it here, and I've done that with min_rainfall, start_day, and end_day below.